packages/amplify-graphql-auth-transformer/src/graphql-auth-transformer.ts

```diff
@@ -416,45 +419,46 @@ Static group authorization should perform as expected.`,
       }
     };
     // default model operations
-    addServiceDirective(ctx.output.getQueryTypeName()!, 'read', modelConfig?.queries?.get);
-    addServiceDirective(ctx.output.getQueryTypeName()!, 'read', modelConfig?.queries?.list);
-    addServiceDirective(ctx.output.getQueryTypeName()!, 'read', modelConfig?.queries?.sync);
+    addServiceDirective(ctx.output.getQueryTypeName()!, 'get', modelConfig?.queries?.get);
+    addServiceDirective(ctx.output.getQueryTypeName()!, 'list', modelConfig?.queries?.list);
+    addServiceDirective(ctx.output.getQueryTypeName()!, 'list', modelConfig?.queries?.sync);
```

Comment:
I think this is correct, but just double checking - is `'list'` correct here for sync queries? Same question for some of the subscriptions below.

Reply:
yes, subscriptions and sync queries are tied to list operation